notation.go

```diff
@@ -203,6 +206,18 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, re
 
 			// verify each signature
 			outcome, err := verifier.Verify(ctx, artifactDescriptor, sigBlob, opts)
+			// process verification items
+			for _, result := range outcome.VerificationResults {
+				if result.Error == nil {
+					continue
+				}
+				switch result.Action {
+				case trustpolicy.ActionLog:
+					logger.Warnf("verification failed on %v validation for signature %v but set to pass by verification action \"logged\". Reason: %v", result.Type, sigManifestDesc.Digest, result.Error)
```

Comment:
Log which signature is being processed, when we actually pull them. Something like

```
Processing signature: sha256@...
   integrity validation failed but is set to "logged". Failure reason:...
   authenticity validation failed but is set to "logged". Failure reason:...
  
```

```suggestion
					logger.Warnf("%v validation failed but is set to \"logged\". Failure reason: %v", result.Type, result.Error)
```

Reply:
Notation-go may be integrated to a service, so it could run in parallel and generate log out of order. If we show  the log in multiple line, it is hard to find the related log in that scenario and unfriendly for scripting.

Reply:
Update: adopted your suggestion.
The parallel scenario can be solved by notation-go user by added a *request-id* field in each log.

Reply:
The way to handle that is take logger from caller of the method and emit the logs in that logger, now its the caller needs to worry about how they want to show logs.